internal/core/domain/trade_service.go

```diff
@@ -72,6 +73,13 @@ func (t *Trade) Accept(
 	t.SwapAccept.Message = swapAcceptMsg
 	t.Timestamp.Accept = uint64(time.Now().Unix())
 	t.PsetBase64 = psetBase64
+
+	p, err := pset.NewPsetFromBase64(psetBase64)
+	if err != nil {
+		return false, err
+	}
+	t.TxID = p.UnsignedTx.TxHash().String()
```

Comment:
can we add an ad hoc method in `transactionutil` pkg that does this? Just to avoid directly importing go-elements.

Reply:
Done